modules/gdscript/language_server/gdscript_language_server.h

```diff
@@ -44,6 +44,7 @@ class GDScriptLanguageServer : public EditorPlugin {
 	bool thread_running = false;
 	bool started = false;
 	bool use_thread = false;
+    String host = "127.0.0.1";
```

Comment:
Indentation should use tabs, not spaces.

Reply:
Sorry about that, I'll make sure I get this setup